src/features/home/courses/courses.scss

```diff
@@ -47,71 +47,52 @@
         }
 
         display: flex;
-        flex-direction: row;
         align-items: center;
         justify-content: space-between;
 
         width: 100%;
         margin-top: 24px;
 
         & .icon-container {
-          position: relative;
+          @include media-mobile-landscape {
+            gap: 5px;
+            padding: 10px;
+
+            & .icon-lang {
+              display: inline-block;
+              font-size: 14px;
+              color: $color-secondary-gray600;
+            }
+          }
 
-          overflow: visible;
           display: flex;
-          flex-flow: row nowrap;
-          flex-shrink: 0;
+          flex-wrap: wrap;
           gap: 0;
```

Comment:
Why do you need it here?

Reply:
Oh, I needed that in one of the language placement options)